test/queries_generator/aws_queries.txt

```diff
@@ -0,0 +1,10 @@
+select lower(lower('  %%AbCdEfGhIjKlMnOpQrStUvWxYz##  ')) from s3object;
+select to_timestamp('1999-11-04T20:27:03.479340Z') from s3object;
+select date_add(day, 8, to_timestamp('1982-02-03T00:20:12.42831Z')) from s3object;
+select substring(upper('  %%AbCdEfGhIjKlMnOpQrStUvWxYz##  '), cast((avg(cast(_3 as int)-cast(_2 as int)) - 3) as int), min(cast(_1 as int)-cast(_1 as int)) + 7) from s3object;
```

Comment:
:+1: 
this kind of query is unique, since its complex and nested, in such case there is a true need for AWS results to compare.